app/femr/common/models/UserItem.java

```diff
@@ -141,3 +143,12 @@ public boolean equals(final Object obj){
         return true;
     }
 }
```

Comment:
@zanni022: Looks like this extra closing bracket got left behind during your rebase